src/main/resources/org/springframework/integration/kafka/config/spring-integration-kafka-2.1.xsd

```diff
@@ -188,6 +188,14 @@
 					<xsd:union memberTypes="listenerMode xsd:string"/>
 				</xsd:simpleType>
 			</xsd:attribute>
+			<xsd:attribute name="payload-type" type="xsd:string">
```

Comment:
We can't modify XSD already. 2.1 version is out. We can start 2.2 or just live without xml support further. Other thought is SI 5.0 with this Kafka module. 
Hi, btw !